workers/unity/Packages/io.improbable.gdk.core/Collections/MessageList.cs

```diff
@@ -31,11 +29,21 @@ public MessageList()
 
         public ref readonly T this[int index] => ref items[index];
 
+        public MessagesSpan<T> Slice()
+        {
+            return new MessagesSpan<T>(this, 0, Count);
+        }
+
+        public MessagesSpan<T> Slice(int index, int count)
+        {
+            return new MessagesSpan<T>(this, index, count);
```

Comment:
Do we want any assertions about `index` or `count` here?

Reply:
do we want it?

Reply:
I mean, I do 😉 (in a debug/development build). I think @zeroZshadow did this but didn't comment/resolve this

Reply:
ah, yeah. Now I see it. Github was giving me an outdated view ;)